lib/facter.rb

```diff
@@ -346,6 +346,8 @@ def value(user_query)
     # @api public
     def values(options, user_queries)
       init_cli_options(options, user_queries)
+      Options[:show_legacy] = true
```

Comment:
`show_legacy` should not be required. If the user queries specifically for a set of facts, Facter will automatically search in legacy fact as well.
e.g.

```ruby
require 'facter'

puts Facter.values({},["uptime"])
```

`uptime` is a legacy fact, and the result is 
```
{"uptime"=>"6 days"}
```